Makefile

```diff
@@ -74,6 +74,7 @@ crds.clean:
 	@$(OK) cleaned generated CRDs
 
 generate: crds.clean
+	go generate ./...
```

Comment:
Ah @pdettori this line is causing the issue. `go generate` is run implicitly, so no need add it here. This is causing us to generate --> clean --> generate again

Reply:
Good catch @hasheddan ! Sorry for that, at some point I might have modified it since I was having issues with generation and did not put it back. Thanks for helping with this.

Reply:
sure thing :)

Reply:
that was it, @hasheddan, the e2e tests and the CI passed. It is ready for review now.